### PR TITLE
ODC-5076-Updated knative-automation framework setup

### PR DESCRIPTION
**Pull request by makambalaji**
**Description**:
This PR is mainly created to setup the framework in knaitve-plugin folder
Jira Id: https://issues.redhat.com/browse/ODC-5076

Please Note: This scripts are not integrated with CI
Add the below line to scripts block in frontend/package.json file

> "test-cypress-knative": "cd packages/knative-plugin/integration-tests && ../../../node_modules/.bin/cypress open --env openshift=true",

Execute the below command "yarn run test-cypress-knative" on fronted folder

Test Results:
No. of smoke test cases in this pr : 1 [Execution Time: - 74 sec]
No. of regression test cases in this pr: [Execution Time: - sec]
No. of test cases in this pr: [Execution Time: - sec]

image

### CODE REVIEW

File: frontend/packages/knative-plugin/integration-tests/features/Knative/event-sources-sink-to-URI.feature [hunk 1]
```diff
@@ -4,7 +4,7 @@ Feature: Event Sources can able sink to URI as well as Resource
 
     Background:
         Given user has installed OpenShift Serverless Operator
-        And user is at Developer perspective
+        And user is at developer perspective
```
Comment (gajanan-more) [inline frontend/packages/knative-plugin/integration-tests/features/Knative/event-sources-sink-to-URI.feature:7]:
```suggestion
        And user is at Developer perspective
```

**Review by gajanan-more — CHANGES_REQUESTED** [review]
There are few file names with underscore and epic number. Can you please update them as well?

File: frontend/packages/dev-console/integration-tests/features/getting-started/sample-card_Add_page.feature [hunk 2]
```diff
@@ -2,7 +2,7 @@ Feature: Create Sample Application
     As a user I want to create the Sample Application from +Add page
 
 Background:
-    Given user is in Developer perspecitve
+    Given user is in Developer perspective
     And open project namespace
```
[the remark this answers is not in the document]
Comment (makambalaji) [reply frontend/packages/dev-console/integration-tests/features/getting-started/sample-card_Add_page.feature:6]: updated the statement, but this pr is not meant for reviewing this feature files

File: frontend/packages/dev-console/integration-tests/features/helm/helm_feature_flag_ODC-5024.feature [hunk 3]
```diff
@@ -3,7 +3,7 @@ Feature: Feature flag for Helm
```
[the remark this answers is not in the document]
Comment (makambalaji) [reply frontend/packages/dev-console/integration-tests/features/helm/helm_feature_flag_ODC-5024.feature:3]: updated

File: frontend/packages/dev-console/integration-tests/features/getting-started/getting-started -to-dev-perspective.feature [hunk 4]
```diff
@@ -0,0 +1,11 @@
+Feature: Login to developer perspective
```
Comment (gajanan-more) [inline frontend/packages/dev-console/integration-tests/features/getting-started/getting-started -to-dev-perspective.feature:1]: Do we still require this scenario? 
Also there is an extra space in the file name.

File: frontend/packages/dev-console/integration-tests/features/getting-started/getting-started -to-dev-perspective.feature [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+              As a user I want to land on developer perspective when login to openshift console as a developer
+
+
+        @regression
+        Scenario: Developer perspective display on login to open shift application using developer credentials
```
Comment (gajanan-more) [inline frontend/packages/dev-console/integration-tests/features/getting-started/getting-started -to-dev-perspective.feature:6]:
```suggestion
        Scenario: Developer perspective display on login to OpenShift application using developer credentials
```

**Review by gajanan-more — CHANGES_REQUESTED** [review]

**Review by gajanan-more — APPROVED** [review]
Screenshot from 2021-02-10 15-36-24

It's working as expected. 

/lgtm

**Pull request merged**